pkg/config/config.go

```diff
@@ -17,36 +17,39 @@ var Cfg = new()
-	DBName              string
-	DBPass              string
-	DBPort              int
-	DBUser              string
-	DevelopmentMode     bool             // Indicates if running in local development mode.
-	Features            featureFlags     // Enable or disable features.
-	Federation          federationConfig // Federated search configuration.
-	HttpPort            int
-	PlaygroundMode      bool   // Enable the GraphQL Playground client.
-	PodNamespace        string // Kubernetes namespace where the pod is running.
-	QueryLimit          uint   // The default LIMIT to use on queries. Client can override.
-	RelationLevel       int    // The number of levels/hops for finding relationships for a particular resource
-	SlowLog             int    // Logs when queries are slower than the specified time duration in ms. Default 300ms
+	HubName                  string //Display Name of the cluster where ACM is deployed
+	API_SERVER_URL           string // address for Kubernetes API Server
+	AuthCacheTTL             int    // Time-to-live (milliseconds) of Authentication (TokenReview) cache.
+	SharedCacheTTL           int    // Time-to-live (milliseconds) of common resources (shared across users) cache.
+	UserCacheTTL             int    // Time-to-live (milliseconds) of namespaced resources (specifc to users) cache.
+	ContextPath              string
+	DBHost                   string
+	DBMinConns               int32 // Overrides pgxpool.Config{ MinConns } Default: 0
+	DBMaxConns               int32 // Overrides pgxpool.Config{ MaxConns } Default: 10
+	DBMaxConnIdleTime        int   // Overrides pgxpool.Config{ MaxConnIdleTime } Default: 30 min
+	DBMaxConnLifeTime        int   // Overrides pgxpool.Config{ MaxConnLifetime } Default: 60 min
+	DBMaxConnLifeJitter      int   // Overrides pgxpool.Config{ MaxConnLifetimeJitter } Default: 2 min
+	DBName                   string
+	DBPass                   string
+	DBPort                   int
+	DBUser                   string
+	DevelopmentMode          bool             // Indicates if running in local development mode.
+	Features                 featureFlags     // Enable or disable features.
+	Federation               federationConfig // Federated search configuration.
+	HttpPort                 int
+	PlaygroundMode           bool   // Enable the GraphQL Playground client.
+	PodNamespace             string // Kubernetes namespace where the pod is running.
+	QueryLimit               uint   // The default LIMIT to use on queries. Client can override.
+	RelationLevel            int    // The number of levels/hops for finding relationships for a particular resource
+	SlowLog                  int    // Logs when queries are slower than the specified time duration in ms. Default 300ms
+	SubscriptionPollInterval int    // Number of seconds between subscription polls
+	SubscriptionPollTimeout  int    // Minutes a subscription will stay open before timeout
```

Comment:
Let's try to keep the time units consistent.
Other time units are in ms, at least these 2 should be the same.

Reply:
Maybe call it `SUBSCRIPTION_REFRESH_INTERVAL` so it still works when we change the implementation.